### ISSUE TITLE
Pre-tain the encoder to learn the pysiological video patterns

### ISSUE THREAD

**Issue by PeterhDynocardia** [poojar25/Distraction1#2]
There should be two phases.

Pre-train model with self-supervised signal (denoising auto-encoder) 

Train model to predict IAH for each time stamp. (where the weight of encoder is frozen. It only generates attention masks) The LSTM will be trianed

**Comment by PeterhDynocardia** [comment]
Please read  [README for pretaining](https://github.com/poojar25/Distraction1/blob/pytorch_implementation_pretraining/Benefit_of_Distraction-master1/get_Dyno/training/README.md)
